packages/viewer-react/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@vertexvis/viewer-react",
+  "version": "0.6.0",
+  "description": "React bindings for the Vertex Viewer SDK.",
+  "author": "Dan Schultz <dan.schultz@vertexvis.com>",
+  "homepage": "https://github.com/Vertexvis/vertex-web-sdk#readme",
+  "bugs": {
+    "url": "https://github.com/Vertexvis/vertex-web-sdk/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Vertexvis/vertex-web-sdk.git"
+  },
+  "license": "ISC",
+  "main": "./dist/bundle.cjs.js",
+  "module": "./dist/bundle.esm.js",
+  "typings": "./dist/index.d.ts",
+  "files": [
+    "dist/*"
+  ],
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org",
+    "access": "public"
+  },
+  "scripts": {
+    "clean": "rm -fr ./dist && mkdir ./dist",
+    "prebuild": "yarn clean",
+    "build": "rollup --config ./rollup.config.js",
+    "format": "yarn lint --fix",
+    "lint": "eslint --ext .ts,.tsx,.js,.jsx --ignore-path ../../.gitignore --ignore-pattern 'src/generated' .",
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
+  "dependencies": {
+    "@vertexvis/viewer": "^0.6.0"
+  },
+  "devDependencies": {
+    "@types/react": "^16.9.46",
+    "@types/react-dom": "^16.9.8",
+    "@vertexvis/eslint-config-vertexvis-typescript": "0.2.2",
+    "@vertexwebsdk/build": "^0.6.0",
```

Comment:
Curious, where is this dependency coming from? I can't seem to find it on NPM, but I'm able to install/build the branch just fine

Reply:
It's from a package in the `./internal/build` folder.